### ISSUE TITLE
[BUG]: ERR_OSSL_BAD_DECRYPT on /app-connections request

### ISSUE THREAD

**Issue by gianlucatursi** [activepieces/activepieces#5157]
**Describe the bug**
after updating from 0.28 to 0.29 I get the error on this request: 

`https://endpoint/api/v1/app-connections?limit=99999&projectId=NIqCufVRMnJqw5lR6rYWi`:

*The error:*

```
{
    "statusCode": 500,
    "code": "ERR_OSSL_BAD_DECRYPT",
    "error": "Internal Server Error",
    "message": "error:1C800064:Provider routines::bad decrypt"
}
```

The enviroment is a self-hosted with in memory queue and an external Postgres. All running on an Azure Service Plan with sidecar.

I cannot configure new or old pieces right now. Could you some help me?

**Comment by hugh-codes** [comment]
I get this too running in dev container

**Comment by abuaboud** [comment]
Here’s a more polished version of your message:

This issue occurs because the AP_ENCRYPTION_KEY is either changed or invalid. To ensure the key is valid, I’ve added a check to confirm it's a valid 32-character hexadecimal string. Additionally, the documentation includes tips on how to generate the key, run `openssl rand -hex 16`

To resolve the issue, you can either restore the original key or clear the app_connections table. While this will delete all connections, the rest of your data will remain intact.

Validation for the key has been added in [this PR](https://github.com/activepieces/activepieces/pull/5523) and will be released in version 0.30.0.

Let me know if you have any further questions, feel free to reopen the  issue.

Best regards,
Mo.
